src/uct/sm/cma/cma_pd.c

```diff
@@ -39,6 +39,16 @@ static ucs_status_t uct_cma_query_pd_resources(uct_pd_resource_desc_t **resource
                                   num_resources_p);
 }
 
+static ucs_status_t uct_cma_mem_reg(uct_pd_h pd, void *address, size_t length,
+                        uct_mem_h *memh_p)
+{
+    /* For testing we have to make sure that
+     * memh_h != UCT_INVALID_MEM_HANDLE
+     * otherwise gtest is not happy */
+    *memh_p = (void *) 0xdeadbeef;
```

Comment:
maybe add UCS_STATIC_ASSERT(0xdeadbeef != UCT_INVALID_MEM_HANDLE)?
you never know..